ray-operator/controllers/ray/raycluster_controller_test.go

```diff
@@ -861,6 +861,47 @@ var _ = Context("Inside the default namespace", func() {
 		})
 	})
 
+	Describe("Suspend RayCluster worker group with Autoscaler enabled", Ordered, func() {
```

Comment:
A unit test seems sufficient if the validation happens at the very beginning of the reconciliation, so no other logic is involved.

Reply:
I think this test case could still be useful when we later add the support to suspend worker groups in an autoscaler-enabled cluster. Do you think we should delete it for now?

Reply:
makes sense